recipe/meta.yaml

```diff
@@ -27,7 +26,7 @@ requirements:
     - setuptools
```

Comment:
setuptools needs to be >= 47: https://github.com/sqlalchemy/alembic/blob/main/setup.py#L5

Reply:
updated the pinnings for `Setuptools`:
https://github.com/AnacondaRecipes/alembic-feedstock/blob/a31bfe5ed4acbbb9916a0ed667603597bae149ad/recipe/meta.yaml#L26